test/second_q/algorithms/ground_state_solvers/test_groundstate_eigensolver.py

```diff
@@ -150,41 +149,6 @@ def test_aux_ops_reusability(self):
             frozenset(a.to_list()) == frozenset(b.to_list()) for a, b in zip(aux_ops, aux_ops_copy)
         )
 
-    def test_list_based_aux_ops(self):
```

Comment:
Is it necessary that we remove this unittest already? I don't have a strong opinion on this, just want to clarify :+1:

Reply:
The same test already exists in the deprecated code. Since now the deprecation msg. only shows twice, the second time the test ran, it failed because the msg wouldn't show again (already shown in the first test). I could have found a way to fix it but didn't think necessary to have the exact same test in two places.

Reply:
Ah I didn't pay enough attention, so I didn't notice that this was the `second_q` file... All good :slightly_smiling_face: